packages/margined_common/src/asset.rs

```diff
@@ -0,0 +1,255 @@
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+use std::fmt;
+
+// use crate::factory::PairType;
+// use crate::pair::QueryMsg as PairQueryMsg;
+// use crate::querier::{query_balance, query_token_balance, query_token_symbol};
+use cosmwasm_std::{
+    to_binary, Addr, Api, BankMsg, Coin, CosmosMsg, MessageInfo, QuerierWrapper, StdError,
+    StdResult, Uint128, WasmMsg,
+};
+use cw20::Cw20ExecuteMsg;
+// use terra_cosmwasm::TerraQuerier;
+
+/// UST token denomination
+pub const UUSD_DENOM: &str = "uusd";
+/// LUNA token denomination
+pub const ULUNA_DENOM: &str = "uluna";
```

Comment:
maybe this gets removed at a later date? 

I'm assuming we aren't going to use or accept UST/Luna